src/Numerics/FVMethods/FVModel_kernels.jl

```diff
@@ -91,7 +91,62 @@ See [`BalanceLaw`](@ref) for usage.
         #    | --------------- | --------------- | --------------- | -----> z
         #    | bot⁻   e⁻  top⁻ | bot    e    top | bot⁺  e⁺   top⁺ |
         
-        for ev in 1:nvertelem
+        ### TODO PERIODIC
+
+        # The first element on the bottom
```

Comment:
I'd hope we could unify the interior and boundary faces, much like we do for DG. You will also need to read the specific boundary condition out of the arrays.